README.md

```diff
@@ -1,9 +1,14 @@
 # WhoWeAre
-Documents about Innovationsverbund Öffentliche Gesundheit (InÖG)
+Documents about Innovationsverbund Öffentliche Gesundheit (InÖG) (loosely translated: Public healt innovation network).
 
 ## What one can find in here
 
-Here we put documents about the Innovationsverbund Öffentliche Gesundheit. There is a lot of German involved, as we are an association based in Germany. We are currently working on English versions. Our statutes will always be in German due to legal reasons.
+Here we put documents about and projects the Innovationsverbund Öffentliche Gesundheit (InÖG).
+
+Our first and most mature project is [https://iris-connect.de](IRIS connect) ([https://github.com/iris-connect/](GitHub)), the e2e-encrypted data interface in and between public health agencies.
+The first use case of this base architecture is transfer of contact tracing data, more is planned and specified at the moment. If you're looking for a place to contribute, this would be a good project to start with.
```

Comment:
IRIS is currently operational in the following german states: North Rhine-Westphalia, Saxony, Thuringia, Hesse and of late parts of Berlin.